src/pages/events/summit-event-list-page.js

```diff
@@ -242,10 +244,14 @@ class SummitEventListPage extends React.Component {
         });
     }
 
+    handleTermChange(term) {
+        this.setState({...this.state, searchTerm: term})
```

Comment:
you are now using the search term on state , but its also on reducer, lets still use the reducer please create an action to set it
regards